src/main/webapp/app/exercises/text/participate/text-editor.component.html

```diff
@@ -61,7 +61,18 @@
                     <jhi-text-result [result]="result"></jhi-text-result>
                     <hr />
                 </div>
-
+                <div class="col-md-12 col-lg-10 col-xl-8 feedback-text mt-5 alert alert-info" *ngIf="unreferencedFeedback">
+                    <b>
+                        <fa-icon [icon]="['far', 'comment-dots']"></fa-icon>&nbsp;<span jhiTranslate="artemisApp.assessment.unreferencedFeedback">
+                            Feedback for Missing Elements</span
+                        >:</b
+                    >
+                    <br />
+                    <div *ngFor="let feedback of unreferencedFeedback">
```

Comment:
Should we rather show a box for each element?

Reply:
I am open to that! I did not put much thinking in the UI to be honest and just followed the general feedback style to keep the view looking consistent but if we go for a different style for unreferenced feedback then I don't mind and can adjust the code accordingly ^^

Reply:
@jpbernius what about a table view just like for modeling exercises?
![image](https://user-images.githubusercontent.com/50635005/82842260-21c6f400-9ed9-11ea-8e5e-95ee0ed9b1a3.png)
Or do you suggest this same style (blue feedback box) but just for each feedback separately?
I feel like the table might be more compact and view space efficient

Reply:
- For text exercises, we do not have such a table, so this would be a new element. So I would follow this "alert style" boxes to keep it consistent. Maybe we can tinker with the margins to make it more space-efficient if this is a concern.
- For modeling, maybe this is an option as the interface is different here.
- But I see how we want to have one UI for this only.

Should we move this to a follow-up? I feel like shipping this in its current form is an improvement, but we should follow up quickly with a new version.

Reply:
Sure!